docs/tsg_pvs.md

```diff
@@ -214,7 +214,24 @@ kubectl apply -f https://raw.githubusercontent.com/kmova/bootstrap/master/gke-op
 
 You can also exec into this application to check the content, retrieve the files, or use the application to check the content. 
 
+## Jiva repliac pods consuming high memory utilisation and warnings in the logs
 
+Where OpenEBS older versions,Jiva replica pods are consuming high memory and showing following error messages in all replica pods happens in every minute 
```

Comment:
Where = In
versions, = versions, <space>
showing = displays
in all replica pods. This happens every minute.

Reply:
Done